sklearn/linear_model/tests/test_logistic.py

```diff
@@ -947,7 +947,8 @@ def test_warm_start():
                                              solver=solver,
                                              random_state=42, max_iter=100,
                                              fit_intercept=fit_intercept)
-                    clf.fit(X, y)
+                    with ignore_warnings():
```

Comment:
- first fit:
`ConvergenceWarning` for SAG solver
- second fit:
`ConvergenceWarning` for SAG solver and `UserWarning` for Newton-CG solver (which actually should be a `ConvergenceWarning`)